recipe/meta.yaml

```diff
@@ -1,14 +1,16 @@
 {% set name = "editdistance" %}
-{% set version = "0.5.3" %}
+{% set version = "0.6.1" %}
 
 package:
-  name: '{{ name|lower }}'
-  version: '{{ version }}'
+  name: {{ name|lower }}
+  version: {{ version }}
 
 source:
-  fn: '{{ name }}-{{ version }}.tar.gz'
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  'sha256': '89d016dda04649b2c49e12b34337755a7b612bfd690420edd50ab31787120c1f'
+  url: https://github.com/roy-ht/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: bc58aac72b82bcc22f15d70423990a568c28fda32392fa0eeba66a288e7b5a19
+  patches:
+    # add python 3.11 compat patch: https://github.com/roy-ht/editdistance/commit/5c81574c1ea9538a4d9c97d1f9c28cc10d67980f
+    - patches/0001-cythonize.patch
 
 build:
   number: 0
```

Comment:
Skip py<36, https://github.com/roy-ht/editdistance/blob/cd61f6bff3e0e407f5e744756b45e88c43edd113/setup.py#L40

Reply:
addressed, thank you